src/app/unit-host/unit-host.component.html

```diff
@@ -25,6 +25,12 @@
     </label>
     <button *ngIf="displayContinueButton()" (click)="sendVopContinueCommand()" title="vopStopCommand">Continue</button>
     <button *ngIf="displayStopButton()" (click)="sendVopStopCommand()" title="vopStopCommand">Stop</button>
+    <select (change)='sendVopNavigationDeniedNotification($event)' style="padding: 1px;" title="Deny Navigation">
```

Comment:
Ich sehe schon, dass das vorher auch so gemacht wurde. Aber Stylings gehören eigentlich in die CSS-Dabei oder nicht?

Reply:
Wenn ich das richtig sehe, brauchst du von dem Event ja nur "target.value". Könnte man da nicht überlegen nur das zu übergeben, statt dem gesamten Event?